microprofile/fault-tolerance/src/test/java/io/helidon/microprofile/faulttolerance/TimeoutTest.java

```diff
@@ -79,7 +94,7 @@ public void testForceTimeoutLoop() {
         try {
             bean.forceTimeoutLoop();        // cannot interrupt
         } catch (TimeoutException e) {
-            assertThat(System.currentTimeMillis() - start, is(greaterThan(2000L)));
+            assertThat(System.currentTimeMillis() - start, is(greaterThanOrEqualTo(2000L)));
```

Comment:
What a chance 🙂

Reply:
Got me curious is there some rounding involved?

Reply:
No, but this is special case where the thread continues to run and cannot be interrupted (note that timeout is 1000). It hit 2000 at least once :)